.github/workflows/acquia-push-code.yml

```diff
@@ -0,0 +1,108 @@
+        required: true
+      ACQUIA_SSH_KEY:
+        description: 'Private SSH key authorised to push to the Acquia repo.'
+        required: true
+      SSH_CONFIG:
+        description: 'SSH configuration'
+        required: true
+      KNOWN_HOSTS:
+        description: 'SSH known hosts'
+        required: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  push-code:
+    name: 'Build artefact and push to Acquia'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Restore build from a previous step.
+        # Only restore if we have a source key.
+        if: inputs.source_key != ''
+        uses: actions/cache@v2
+        id: restore-build
+        with:
+          path: ./*
+          # We are only interested in a direct hit.
+          key: build-${{ github.sha }}
+
+      - name: Abort if we have a source build cache miss.
+        run: exit 1
+        # Only check if we have a source key.
+        if: |
+          inputs.source_key != '' &&
+          steps.restore-build.outputs.cache-hit != 'true'
+
+      - name: Checkout the code.
+        # Only if we do not have a source key.
+        if: inputs.source_key == ''
+        uses: actions/checkout@v3
+
+      - name: Preserve the composer cache.
+        # Only if we do not have a source key.
+        if: inputs.source_key == ''
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          # Save the cache with the hash of the lock files.
+          key: composer-${{ hashFiles('**/composer.lock') }}
+          # If there is no direct hit, restore the latest composer-* cache. This
+          # will first look on the current branch, then on the parent/default
+          # branch.
+          restore-keys: |
+            composer-
+
+      - name: Preserve the npm cache.
+        # Only if we do not have a source key.
+        if: inputs.source_key == ''
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.npm
+            ~/.nvm
+          # Save the cache with the hash of the lock files.
+          key: npm-${{ hashFiles('**/package-lock.json') }}
+          # If there is no direct hit, restore the latest npm-* cache. This will
+          # first look on the current branch, then on the parent/default branch.
+          restore-keys: |
+            npm-
+
+      - name: Set up the SSH key and configuration.
+        uses: shimataro/ssh-key-action@v2.3.1
+        with:
+          key: ${{ secrets.ACQUIA_SSH_KEY }}
+          config: ${{ secrets.SSH_CONFIG }}
+          known_hosts: ${{ secrets.KNOWN_HOSTS }}
+
+      - name: Deploy the artifact for branches
+        if: startsWith(github.ref, 'refs/heads/')
+        run: |
+          ./vendor/bin/blt artifact:deploy --commit-msg "[BUILD ${COMMIT_SHA:0:7}] $COMMIT_MSG" --branch "${COMMIT_REF:11}-build" --no-interaction --verbose
+          echo "⚡ Branch ${COMMIT_REF:11}-build pushed to Acquia!" >> $GITHUB_STEP_SUMMARY
```

Comment:
Totes feel like this should be 💧 rather than ⚡ (which I used for Pantheon because, well, Pantheon). :D

Reply:
So I've changed it to a generic BLT action (in case we ever want that elsewhere and to differentiate from a non BLT build and push that could go to acquia)... Feel free to suggest a new icon, not my forte!

Reply:
BLT should be 🤮

Or 🥪